src/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ add_executable(xenomods
 		xenomods/modules/PlayerMovement.cpp
 		xenomods/modules/BdatRandomizer.cpp
 		xenomods/modules/FrameratePatch.cpp
+        xenomods/modules/AppletRedirect.cpp
```

Comment:
Try to line these up with the rest, I know the clang-format doesn't like that though